### DISCUSSION TITLE
Strange behavior with cape-dabbrev and corfu-auto enabled

### DISCUSSION THREAD

**Discussion by catweazle9** [minad/cape#137] [Q&A] [answered]
I have cape-dabbrev enabled:   (add-hook 'completion-at-point-functions #'cape-dabbrev) 
.. and corfu-auto enabled with the default corfu-auto-prefix value of 3, and also global-corfu-mode enabled.

My scratch buffer contains the following 2 words:
zeville
zevolt

In another buffer, if I type 'zev' the completions list pops up automatically as expected but contains only 1 item 'zeville', 
If I then Ctrl-g and run M-x C-u C-u dabbrev-completion   (using Ctrl-u twice to force dabbrev-completion to search all buffers)
.. corfu now correctly pops up its menu with both candidates.

A secondary problem:
- Clear all text in the second (non-scratch) buffer
- Type 'zevi' quickly (quick enough so the popup isn't displayed after typing 3 chars) - no popup appears at all after typing the 4th character (it again does appear if I manually run dabbrev-completion with 16 as the prefix arg). Same problem if I quickly type 'zevo'.

Could this be a bug in the cape-dabbrev implementation?

Thanks, John.

**Comment by minad** [comment]
Maybe this is the same problem as in https://github.com/minad/cape/discussions/134?

**Reply by catweazle9** [reply]
I don't know.
Can you repro the issue as I've described it, with exactly those words in buffer A while running cape-dabbrev from buffer B?

**Reply by catweazle9** [reply]
@minad yes, I think it is the same issue as #134 .

I may have a simpler repro:
1. Disable corfu-auto (the problem occurs even without it)
2. Enter `zeville ` in the \*scratch\* buffer
3. In a different buffer type `zev M-x cape-dabbrev ` - it completes as expected
4. Delete that line and type `zevi M-x cape-dabbrev` - no completion. If you then run `C-u C-u M-x dabbrev-completion` that does work

**Reply by minad** [reply, marked as answer]
Okay, so it is not an issue? You can set `cape-dabbrev-min-length` to 0.

**Reply by catweazle9** [reply]
Yes, that seems to fix it.
I wasn't sure how `cape-dabbrev-min-length` interacts with `corfu-auto-prefix`, but I haven't yet seen any issues.
Thanks!
